packages/admin-ui/ui/src/hooks/use-build-timeline.tsx

```diff
@@ -321,6 +329,7 @@ export const useBuildTimeline = (orderId: string) => {
         items: event.items.map((item) => getLineItem(allItems, item.item_id)),
```

Comment:
thought: I think that `getLineItem` here might just be taking the order item with the item quantity from the order instead of using the fulfillment items quantity - my guess is that this is what creates the discrepancies.

Reply:
Ah I get you now, let me try and replicate.

Reply:
Ok, fixed, added a new getter specifically for fulfilment items, that passes in the whole item instead of just the id and makes use of the quantity off that item. Used it in "fulfiled" and "shipped" events, seems in order now 👍.